fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/LaoDetailActivity.java

```diff
@@ -226,6 +229,17 @@ public void setupSocialMediaButton() {
     socialMediaButton.setOnClickListener(v -> mViewModel.openSocialMedia());
   }
 
+  public void setupDigitalCashButton() {
+    Button digitalCashButton = findViewById(R.id.tab_digital_cash);
+    digitalCashButton.setOnClickListener(
+        new View.OnClickListener() {
+          public void onClick(View v) {
+            Toast.makeText(LaoDetailActivity.this, "Button for Transaction", Toast.LENGTH_LONG)
+                .show();
+          }
```

Comment:
You can simplify this with a lambda :

```suggestion
         v -> Toast.makeText(LaoDetailActivity.this, "Button for Transaction", Toast.LENGTH_LONG).show()
```

And as stated in the last PR, you should use `string.xml` values when text is shown to the user.